extensions/cpp_region/PyRegion.hpp

```diff
@@ -55,6 +57,9 @@ namespace nupic
     void serialize(BundleIO& bundle);
     void deserialize(BundleIO& bundle);
 
+    virtual void write(capnp::AnyPointer::Builder& proto) const override;
```

Comment:
Good idea. I don't want to get into too much detail since there are a million places where this exists. Ultimately uses should understand the difference at the high level since I don't think we can have a full explanation at every single entry point to serialization.